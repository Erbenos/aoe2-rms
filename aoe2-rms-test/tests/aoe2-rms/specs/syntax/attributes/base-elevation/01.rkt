#lang aoe2-rms
; Create a palm desert land with elevation 2.

<LAND-GENERATION>
(create-land
 (terrain-type 'PALM_DESERT)
 (number-of-tiles 128)
 (base-elevation 2))

<ELEVATION-GENERATION>
