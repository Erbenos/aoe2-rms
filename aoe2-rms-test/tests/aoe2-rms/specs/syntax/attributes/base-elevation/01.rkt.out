<LAND_GENERATION>
create_land {
  terrain_type PALM_DESERT
  number_of_tiles 128
  base_elevation 2
}
<ELEVATION_GENERATION>
